config/resource.go

```diff
@@ -62,6 +62,7 @@ type Component struct {
 
 	Namespace     resource.Namespace           `json:"namespace"`
 	Type          resource.SubtypeName         `json:"type"`
+	API           resource.Subtype             `json:"api"`
```

Comment:
this is probably fine but use `json"-"` with a comment that it's an experimental field